packages/phone-number-privacy/combiner/src/index.ts

```diff
@@ -1,12 +1,13 @@
 import * as functions from 'firebase-functions'
 import logger from './common/logger'
 import { handleGetContactMatches } from './match-making/get-contact-matches'
-import { handleGetBlindedMessageForSalt } from './salt-generation/get-salt'
+import { handleGetDistributedBlindedMessageForSalt } from './salt-generation/get-threshold-salt'
 
-// EG. curl -v "http://localhost:5000/celo-phone-number-privacy/us-central1/getBlindedSalt" -H "Authorization: 0xdaf63ea42a092e69b2001db3826bc81dc859bffa4d51ce8943fddc8ccfcf6b2b1f55d64e4612e7c028791528796f5a62c1d2865b184b664589696a08c83fc62a00" -d '{"hashedPhoneNumber":"0x5f6e88c3f724b3a09d3194c0514426494955eff7127c29654e48a361a19b4b96","blindedQueryPhoneNumber":"n/I9srniwEHm5o6t3y0tTUB5fn7xjxRrLP1F/i8ORCdqV++WWiaAzUo3GA2UNHiB","account":"0x588e4b68193001e4d10928660aB4165b813717C0"}' -H 'Content-Type: application/json'
-export const getBlindedSalt = functions.https.onRequest(async (request, response) => {
```

Comment:
Thinking about the rollout.. It'd probably easiest to support backward compatibility in the server over the mobile client. We can leave both methods in here and once the mobile client is updated to use `getDistributedBlindedSalt` we can clean up the old `getBlindedSalt` method. WDYT?

Reply:
ya -- that's a possibility to have the mobile client call the combiner directly with the old method...🤔

i'll look into it as a good transition option

Reply:
alternative rossy suggested was just to stand it up separately and leave the current service in firebase running.  then redirect the mobile client to the new url of the combiner

Reply:
Rossy's idea sounds like less hassle 👍